analyzer/windows/modules/packages/exe.py

```diff
@@ -11,6 +11,28 @@
 class Exe(Package):
     """EXE analysis package."""
 
+    REGKEYS = [
+        [
+            HKEY_LOCAL_MACHINE,
+            "Software\\Microsoft\\Security Center",
+            {
+                # "Would you like Internet Explorer as default browser?"
```

Comment:
that's true, but I'd like to have this bit automated aswell (I guess you might also) :) It makes more sense and buys me 30 min if I need to rebuild whole sandbox from scratch ;)